wiki/plugins/images/settings.py

```diff
@@ -12,5 +12,3 @@
 ANONYMOUS = getattr(django_settings, 'WIKI_ATTACHMENTS_ANONYMOUS', False)
 
 SLUG = 'images'
-
-APP_LABEL = 'wiki'
```

Comment:
Right, all the stuff I removed was around making the app_label something besides wiki. Now just the default is used.